Gems/ScriptCanvas/Code/Editor/View/Windows/MainWindow.cpp

```diff
@@ -523,6 +522,18 @@ namespace ScriptCanvasEditor
 
         m_editorToolbar->AddCustomAction(m_validateGraphToolButton);
 
+        // Screenshot
+        {
+            m_takeScreenshot = new QToolButton();
+            m_takeScreenshot->setToolTip("Captures a full resolution screenshot of the entire graph or selected nodes into the clipboard");
+            m_takeScreenshot->setIcon(QIcon(":/ScriptCanvasEditorResources/Resources/scriptcanvas_screenshot.png"));
```

Comment:
Would it be possible to use svg icons for buttons? (Can be a follow-up task since I see all icons here still use pngs)

Reply:
I considered it, but it would be inconsistent. I can open an issue to convert all existing icons to SVG